doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -311,7 +310,8 @@ Rewriting with Leibniz and setoid equality
         and :n:`@ident__2 = g @ident__1` which without the
         flag would be a cause of failure of :tacn:`subst`.
 
-      Additionally, it prevents a local definition such as :n:`@ident := t` from being
+      Additionally, it prevents a :term:`local definition <context-local definition>`
+      such as :n:`@ident := t` from being
       unfolded which otherwise it would exceptionally unfold in configurations
```

Comment:
```suggestion
      unfolded which otherwise would exceptionally unfold in configurations
```

Reply:
Done.  Note the paragraph wording is still a bit odd, but changing it further is beyond the scope of this PR, which is focused on the tactics chapter, which has been pending for a very long while.